elements/pl-pivot-table/pl-pivot-table-single.mustache

```diff
@@ -0,0 +1,132 @@
+{{#question}}
+        <script>
+            var uuid = "{{uuid}}";
+            var num_row_dropzone = parseInt("{{num_row_dropzone_val}}");
+            var num_index = 1;
+            var is_multicol = "True" === "{{is_multicol}}";
+        </script>
+        <div class="container column-box">
+            <h2>Choose answer column here</h2>
+            {{#column_set}}
+            <div class="row col-set" id="{{uuid}}" data-order="{{order_col}}">
+                {{#column}}
+                <div class="col-md-{{{width}}} col-cell">{{{inner_html}}}</div>
+                {{/column}}
+                {{#is_ellipsis}}
+                <div class="col-md-{{{width}}} col-cell">...</div>
+                {{/is_ellipsis}}
+            </div>
+            {{/column_set}}
+        </div>
+
+        <div class="container index-box">
+            <h2>Choose answer index here</h2>
```

Comment:
same comment about customizing section names

Reply:
This is resolved